CHANGELOG.md

```diff
@@ -6,7 +6,11 @@
 
 _Aug 22, 2022_
 
-A big thanks to the 11 contributors who made this release possible. This release was mostly around 🐛 bug fixes and 📚 documentation improvements.
+A big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- ✨ [icons] Sync the Material Icons (#33988) @michaldudak 
+- Changes in icons' appearances can be seen [here](https://app.argos-ci.com/mui/material-ui/builds/4428])
```

Comment:
```suggestion
- ✨ @michaldudak synced the Material Icons set with the latest from Google (#33988).  
  A couple of icons changed their appearance. See the difference [on this Argos build](https://app.argos-ci.com/mui/material-ui/builds/4428]).
```

Reply:
@hbjORbj looks like this is better than my version. Your call!